Running in wrapper mode for '288'

Running python job.

[name: "CPU-QUOTA"
int_val: 44
int_val: 37
int_val: 87
int_val: 70
int_val: 95
int_val: 53
int_val: 78
, name: "DISK"
int_val: 29
int_val: 29
int_val: 24
int_val: 23
int_val: 16
int_val: 12
int_val: 10
, name: "MEMORY"
int_val: 29
int_val: 18
int_val: 39
int_val: 37
int_val: 16
int_val: 8
int_val: 24
, name: "NET"
int_val: 39
int_val: 38
int_val: 21
int_val: 8
int_val: 8
int_val: 5
int_val: 39
]
The paramters are {u'CPU-QUOTA': array([44, 37, 87, 70, 95, 53, 78]), u'NET': array([39, 38, 21,  8,  8,  5, 39]), u'DISK': array([29, 29, 24, 23, 16, 12, 10]), u'MEMORY': array([29, 18, 39, 37, 16,  8, 24])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
Setting cpu quota to 92500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, NET 194560.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, DISK 24326963.2
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, MEMORY 675000000.0
Recovering the memory utilization
Setting cpu quota to 110000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, NET 199680.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, DISK 24326963.2
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, MEMORY 1087500000.0
Recovering the memory utilization
Setting cpu quota to 217500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, NET 107520.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, DISK 20132659.2
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, MEMORY 1462500000.0
Recovering the memory utilization
Setting cpu quota to 110000 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, NET 199680.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, DISK 24326963.2
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, MEMORY 1087500000.0
Recovering the memory utilization
Setting cpu quota to 217500 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, NET 107520.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, DISK 20132659.2
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, MEMORY 1462500000.0
Recovering the memory utilization
Using 5 trials
INVALID EXPERIMENT TYPE: todo-app

----------------------------------------
Problem running the job:
(<type 'exceptions.SystemExit'>, SystemExit(None,), <traceback object at 0x7f40384aa6c8>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 204, in explore_spearmint
    experiment_results = run_experiment.measure_runtime(workload_config, experiment_trials)
  File "/home/ubuntu/throttlebot/src/run_experiment.py", line 37, in measure_runtime
    exit()
  File "/usr/lib/python2.7/site.py", line 375, in __call__
    raise SystemExit(code)
SystemExit: None
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 15.39 seconds.
